scripts/cloudant/create_databases.py

```diff
@@ -12,6 +12,10 @@
 
     cloudant_instance = authenticate_cloudant_instance(args.username)
 
+    ask_runs = []
+
     for database in args.databases:
         if not cloudant_instance.database_exists(database):
-            cloudant_instance.create_database(database)
+            ask_runs.append(cloudant_instance.create_database(database))
```

Comment:
is the intention to only create the database after asking? it looks like this creates it before calling `run_ask_runs`